lib/handbook/addon/docs/components/osf-dialog/template.md

```diff
@@ -19,7 +20,6 @@ This is a dialog box! Use it for dialog boxes and modal popup things.
 * `@renderInPlace` (optional; default `false`)
     * whether the dialog should be rendered in place (instead of wormhole'd to the app root)
     * forwarded to [ember-wormhole](https://github.com/yapplabs/ember-wormhole#can-i-render-in-place-ie-unwormhole)
-    * good in integration tests, but generally shouldn't be used in real life
```

Comment:
Is there a reason this is removed?

Reply:
this is not needed anymore with this commit https://github.com/CenterForOpenScience/ember-osf-web/pull/814/commits/a0441e7a80d8284165d9e0f1595f5b72de049ce7. basically, osfDialog handles this for the tests use cases